modules/installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+// * installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-nutanix-failure-domains_{context}"]
+= Configuring failure domains
+
+Failure domains improve the fault tolerance of an {product-title} cluster by distributing control plane and compute machines across multiple Nutanix Prism Elements (clusters).
+
+[TIP]
+====
+It is recommended that three failure domains be configured to ensure high-availability.
+====
+
+.Prerequisites
+
+* You have an installation configuration file (`install-config.yaml`).
+
+.Procedure
+
+. Edit the `install-config.yaml` file and add the following stanza to configure the first failure domain:
++
+[source,yaml]
+----
+# ...
+platform:
+  nutanix:
+    failureDomains:
+    - name: <failure_domain_name>
+      prismElement:
+        name: <prism_element_name>
+        uuid: <prism_element_uuid>
```

Comment:
@yanhua121 That's a little strange to me, so should we remove `endpoint` from above field explanation?

Reply:
@sgaoshang @mjpytlak Since the endpoint of the prismElement is optional, and here in the use case of failureDomain, the PE endpoint is not used by the installer so we don't bother to document it. But in the use case of platform.nutanix.prismElements, the PE endpoint is used by the installer to construct the Infrastructure CR manifest. But if you feel we need to document all the optional fields, no matter it is used or not, @mjpytlak You can add the endpoint documentation here and mark it is optional.

Reply:
@yanhua121 Thanks for the explanation, I'm OK with both way then.